.github/workflows/release.yml

```diff
@@ -328,19 +344,32 @@ jobs:
   create-release:
     runs-on: ubuntu-latest
     needs: code-sign-mac-installers
+    env:
+      PLUGIN_TARGET: "/CreateBridgeStable/"
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
     steps:
 
       - name: Download artifact
         uses: actions/download-artifact@v2 # download all the artifacts
 
+      - name: Identify Prerelease
+        # This is a workaround while waiting for create-release action to implement auto pre-release based on tag
+        id: prerelease
+        run: |
+          wget -q -P /tmp https://github.com/fsaintjacques/semver-tool/archive/3.1.0.zip
+          unzip -p /tmp/3.1.0.zip semver-tool-3.1.0/src/semver >/tmp/semver && chmod +x /tmp/semver
+          if [[ $(/tmp/semver get prerel ${GITHUB_REF/refs\/tags\//}) ]]; then echo "::set-output name=IS_PRE::true"; fi
+
```

Comment:
just like in the [arduino-cli](https://github.com/arduino/arduino-cli/blob/8d026eddbfb45ef3ccf025e419e46d9754f38398/.github/workflows/release.yaml#L118-L125) this step is useful to check if a release is actually a prerelease based on the tag (`1.2.0-dev` and `1.2.0-rc1` are both considered prereleases)